### DISCUSSION TITLE
Upgrade fails: Class not found

### DISCUSSION THREAD

**Discussion by thoasty-dev** [livewire/livewire#6368] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.9

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Use php artisan livewire:upgrade
2. It throws an error after the step for "forgetComputed"

### Additional information

The class is somehow not in the repo?

```
Illuminate\Contracts\Container\BindingResolutionException 

 Target class [Livewire\Features\SupportConsoleCommands\Commands\Upgrade\ReplaceEmitWithDispatch] does not exist.
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment]
For some reason, the class is moved to `stubs/ReplaceEmitWithDispatch.php` in this PR #6312

PR: https://github.com/livewire/livewire/pull/6369

@PhiloNL

**Comment by PhiloNL** [comment]
Sorry about that, it should be fixed in the next release. Thanks for the PR @PerryvanderMeer!
